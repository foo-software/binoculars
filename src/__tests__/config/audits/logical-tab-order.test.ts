import auditLogicalTabOrder from '../../../config/audits/logical-tab-order';

describe('logical-tab-order audit', () => {
  it('should match snapshot', () => {
    expect(auditLogicalTabOrder('en')).toMatchSnapshot();
  });
});
