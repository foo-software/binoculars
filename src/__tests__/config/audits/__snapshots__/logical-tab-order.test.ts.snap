// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logical-tab-order audit should match snapshot 1`] = `
Object {
  "description": "Tabbing through the page follows the visual layout. Users cannot focus elements that are offscreen. [Learn more](https://web.dev/logical-tab-order/).",
  "title": "The page has a logical tab order",
}
`;
